final_task/rss_reader/requirements.txt

```diff
@@ -0,0 +1,21 @@
+beautifulsoup4==4.8.1
+cairocffi==1.1.0
+CairoSVG==2.4.2
+cffi==1.13.2
+coverage==4.5.4
+cssselect2==0.2.2
+defusedxml==0.6.0
+html5lib==1.0.1
+Jinja2==2.10.3
+lxml==4.4.1
+MarkupSafe==1.1.1
+nose2==0.9.1
+Pillow==6.2.1
+pycparser==2.19
+Pyphen==0.9.5
+python-dateutil==2.8.1
+six==1.13.0
+soupsieve==1.9.5
+tinycss2==1.0.2
+WeasyPrint==50
+webencodings==0.5.1
```

Comment:
There is too much redundant libs here. Usually only libs that are needed for your application _directly_ specified in this file.

Reply:
Oh, I see. I should not list all of them here even if most of them were installed by libs that I use directly?

Reply:
>Oh, I see. I should not list all of them here even if most of them were installed by libs that I use directly?

Yes. The same for `dependencies` in `setup.py`

Reply:
I get it. Thanks